portfolio/src/main/java/com/google/sps/servlets/Task.java

```diff
@@ -2,13 +2,13 @@
 
 public final class Task {
```

Comment:
It seems to me that "Task" doesn't accurately describe what this class is for. Can you give it a more descriptive name like "Message" or similar?